web/src/stream_ui_updates.js

```diff
@@ -46,6 +46,7 @@ export function initialize_disable_btn_hint_popover(
 export function initialize_cant_subscribe_popover(sub) {
     const $button_wrapper = stream_settings_containers
         .get_edit_container(sub)
+        .parent()
```

Comment:
Can we replace this with a selector that names the class the parent is expected to have? Doing that is likely to be much more robust to breaking again in future refactoring, since grepping for the class name would identify this code depends on that element.